src/control/events/pubsub_test.go

```diff
@@ -234,32 +235,64 @@ func TestEvents_PubSub_Debounce_NoCooldown(t *testing.T) {
 }
 
 func TestEvents_PubSub_Debounce_Cooldown(t *testing.T) {
-	log, buf := logging.NewTestLogger(t.Name())
-	defer test.ShowBufferOnFailure(t, buf)
+	test := func(t *testing.T, iter int) bool {
```

Comment:
minor: Avoid variable names that could conflict with package names. We already have an imported test package. This obviously works OK as-is because the compiler figures it out, but as a general rule we should try to avoid confusion.

Reply:
@mjmac I will address these comments in a follow-on.